content/reference/admin/private_locations/configuration/includes/control-plane.kubernetes.md

```diff
@@ -0,0 +1,64 @@
+```bash
+control-plane {
+  # Control plane token
+  token = "cpt_example_c7oze5djp3u14a5xqjanh..."
+  # Control plane description (optional)
+  description = "my control plane description"
+  # Locations configurations
+  locations = [
+    {
+      # Private location ID, must be prefixed by prl_, only lower alpha,
+      # numeric, underscores and with a max length of 30 characters
+      id = "prl_private_location_example"
+      # Private location description (optional)
+      description = "Private Location on Kubernetes"
+      # Private location type
+      type = "kubernetes"
+      # Namespace (optional, default based on kubernetes configuration)
+      namespace = "gatling"
+      # Certified image configuration
+      image {
+        type = certified
+        java = 17 # Possible values : "8", "11" or 17"
+      }
+      # Custom image configuration
+      # image {
+      #   type = custom
+      #   image = gatlingcorp/frontline-injector:17
+      # }
+      # Clean up finished jobs resources after given time (optional)
+      ttl-after-finished = 10 minutes
+      # Labels of initiated resources (optional)
+      labels {
+        # ExampleKey = ExampleValue
+      }
+      # Annotations of initiated resources (optional)
+      annotations {
+        # ExampleKey = ExampleValue
+      }
+      # Environment variables of initiated pods (optional)
+      environment-variables {
+        # ExampleKey = ExampleValue
+      }
+      # Tolerations (optional)
+      tolerations = [
+        {
+          key = key1
+          operator = Equal
+          # Value is not needed when effect is Exists (optional)
+          value = value1 
+          # An empty effect matches all effects with key (optional)
+          effect = NoSchedule
+        }
+      ]
+    }
+  ]
+}
+```
+
+{{< alert tip >}}
+Kubernetes private locations require the control plane to have access a valid kubeconfig file.
```

Comment:
Actually, you don't need anything if your control plane is run from the cluster. As it will often  be the case, maybe we can invert this alert : If you run control-plane outside the cluster, you need a valid kubeconfig....

Reply:
Yes, but isn't it because kubeconfig is configured (based on role binding) to the pods?